[WIP] Add “include_hidden” checkbox

kvassss opened the pull request:
This checkbox changes the default behavior to exclude backups of hidden files and directories. If this checkbox is checked, mintbackup will add all hidden files and directories to the archive as well

clefebvre commented:
I don't like the way this works UI-wise.

I'd be OK with a button that says "Include all", which adds all of them to the treeview, as if you added them individually with "Add files" and "Add directories".

kvassss commented:
Added a button to add all hidden files and directories from the root directory
Removed GtkViewport, as it interfered with the display of new items in GtkTreeView — not all items appeared, and the remaining ones appeared only after resizing the window or, for example, deleting 1 item from the list.
